vim25/soap/soap.go

```diff
@@ -47,3 +47,15 @@ type Fault struct {
 func (f *Fault) VimFault() types.AnyType {
 	return f.Detail.Fault
 }
+
+func (f *Fault) Error() string {
+	return f.String
+}
+
+func (f *Fault) Unwrap() error {
```

Comment:
Had to do this trick because I cannot use any of the `soap` package types/helpers in `types.go` to do the appropriate type assertions/casting.